### PR TITLE
feat(frontend): team mentors pages

### PR DESCRIPTION
**Pull request by urlDev**
Created team and mentors pages. Created a reusable card component to use in both team and mentors pages. Fixed footer position on different pages. Changed favicon to the new logo. Changed header image with an inspirational image from unsplash. Changed padding for the card under tl;dr. Changed links from a to next/link. For now, keeping the placeholders down until changing the introduction with putting the placeholder under it maybe.

Added active class to page links. Aligned social media icons in cards, the Twitter icon was staying off to the top.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ If you want to tackle an issue:
 
     1. create a fork from iamfreecodecamp repo
     2. create a feature branch
-    3. create a PR with detailed and well documented descriptions about what your doing.
+    3. create a PR to our `develop`branch with detailed and well documented descriptions about what your doing.
```
Comment (evedes) [inline README.md:22]: There's a typo, lacking a space between `develop` and branch

File: components/card.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import { FaTwitter, FaLinkedin } from 'react-icons/fa';
+
+export default function Card({ team }) {
```
Comment (evedes) [inline components/card.js:3]: Maybe team isn't the best variable name to use here? 🤔  Each Card renders a user? person? individual?

File: components/header.js [hunk 3]
```diff
@@ -1,25 +1,38 @@
+import Link from 'next/link';
+import { useRouter } from 'next/router';
 import { FaTwitter } from 'react-icons/fa';
 
+// const links = [
```
Comment (evedes) [inline components/header.js:5]: We can remove this commented out code.

File: pages/mentors.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import Card from '../components/card';
+import Layout from '../components/layout';
+
+const mentors = [
+  {
+    name: 'Eduardo Vedes',
+    location: 'PT',
+    image: '/assets/team/eduardo.jpg',
+    title: 'Software Developer',
```
Comment (evedes) [inline pages/mentors.js:9]: I'd prefer Software Engineer here. It has a more wide touch :)

File: pages/mentors.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    description: 'Self-taught developer focusing on React, Vue and Node'
```
Comment (evedes) [inline pages/mentors.js:10]: Selt-taught developer focused on the JavaScript ecosystem. freeCodeCamp Advocate, Tech Writer.

File: pages/team.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    twitter: 'urlDev',
+    linkedIn: 'can-ural'
+  }
+];
+
+export default function Team() {
+  return (
+    <Layout>
+      <div className="md:grid grid-cols-2 gap-4 mt-8">
+        {teamMembers.map((team) => (
```
Comment (evedes) [inline pages/team.js:29]: Instead of team maybe member? 🤔

File: pages/team.js [hunk 7]
```diff
@@ -0,0 +1,35 @@
+import Card from '../components/card';
+import Layout from '../components/layout';
+
+const teamMembers = [
```
Comment (evedes) [inline pages/team.js:4]: This object is kind of repeated on mentors and team pages. Maybe create a variable in libs or utils folder and export it from there with all the key/values needed for both situations? 🤔

**Review by evedes — COMMENTED** [review]
Code looks great! Just left you some minor comments here and there. Besides that let's try to not commit package-lock.json when we don't add something new to package.json. Also, we're using yarn so you should do `yarn` instead of `npm install` :)

**Comment by urlDev** [comment]
Fixed background color, card responsive sizes. Added custom height for the header image. Created a utils folder for team/mentors variables. Created a card-list component to style the list of cards from there instead of pages. Changed descriptions.

**Review by evedes — APPROVED** [review]

**Pull request merged**